### PR TITLE
Roopchansinghv/osx 64 builds

### PR DESCRIPTION
**Pull request by roopchansinghv**
Trying again to build, but this time, using Conda's environment setup documentation [here](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#macos-and-linux) and [here](https://conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#using-the-compiler-packages), to specify C and C++ compilers in environment to make and cmake.

### CODE REVIEW

File: siemens_to_ismrmrd/meta.yaml [hunk 1]
```diff
@@ -24,4 +26,4 @@ requirements:
     - libxslt=1.1.33 
 
 about:
-  home:
\ No newline at end of file
+  home:
```
Comment (hansenms) [inline siemens_to_ismrmrd/meta.yaml:29]: Let's try to add meta data when we update these packages.

File: gadgetron-python/meta.yaml [hunk 2]
```diff
@@ -8,18 +8,18 @@ source:
...
+    - python>=3.6
     - multimethod>=1.0
     - numpy>=1.5.1
     - ismrmrd-python>=1.9.5
     - pyfftw>=0.11.0
 
 about:
-  home:
\ No newline at end of file
+  home:
```
Comment (hansenms) [inline gadgetron-python/meta.yaml:25]: Let's try to add metadata when we touch up these packages.

**Review by hansenms — CHANGES_REQUESTED** [review]
We should add a git workflow, which builds on macos. If we are unable to get the gadgetron building for now, we can just get the dependencies in by explicitly listing the packages we want to build, e.g.:

```bash
./build.sh -p ismrmrd -p ismrmrd-pyton -p ...
```

So we get the ones we have working built and pushed.

**Comment by roopchansinghv** [comment]
Okay - closing this PR, and opening another with auto-building support packages on macOS.

**Pull request closed without merging**